cards.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cards</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav>
+        <div class="nav1-list">
+      
+            <ul class="topnav">
+      <li class="left"><a href="index.html">Mars UI</a></li>
+      <li class="right"><a href="https://www.linkedin.com/in/anjali-rohira-886a52187/">Contact</a></li>
+      <li class="right"><a href="https://github.com/rohiraanjali/Mars-Ui"
+         target="_blank" rel="noreferrer">
+         <svg class="octicon octicon-mark-github v-align-middle" height="30" viewBox="0 0 16 16" version="1.1" width="30" aria-hidden="true"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"></path></svg>
+       
+        </a>
+        </li>
+      <li class="right"><a href="index.html">Home</a></li>
+          </ul>
+        </div>
+      </nav>
+
+      <div class="sidenav">
+
+        
+        <a href="avatar.html">Avatar</a><br />
+
+        <a href="alert.html">Alert</a><br />
+        <a href="button.html">Buttons</a><br />
+        <a href="image.html">Images</a><br />
+        <a href="input.html">Inputs</a><br />
+        <a href="navigation.html">Navigation</a><br />
+        <a href="modal.html">Modal</a><br />
+        <a href="rating.html">Ratings</a><br />
+        <a href="Grids.html">Grids</a><br />
+        <a href="cards.html">Cards</a><br />
+        <a href="toast.html">Toasts</a><br />
+        <a href="text.html">Text Utilities</a><br />
+        <a href="lists.html">Lists</a><br />
+      </div>
+      <main>
+        <h2 class="main-title">Cards</h2>
+        <p class="body-subtitle">Cards contain content and actions related to a single subject or entity
+        </p>
+        <h3 class="usage-component"><u>Usage</u></h3>
+        <p class="body-subtitle">Cart Card
+        </p>
+        <div class="card1-div">
+          <div class="card1">
```

Comment:
![image](https://user-images.githubusercontent.com/39916455/115251469-37c05180-a148-11eb-81f5-7fe5fdeef32e.png)
card1 is overflowing.